src/CalcManager/UnitConverter.h

```diff
@@ -51,7 +51,7 @@ namespace UnitConversionManager
     // null checks.
     //
     // unitId, name, abbreviation, isConversionSource, isConversionTarget, isWhimsical
-    const Unit EMPTY_UNIT = Unit{ -1, L"", L"", true, true, false };
+    extern __declspec(selectany) const Unit EMPTY_UNIT = Unit{ -1, L"", L"", true, true, false };
```

Comment:
Hey @jefgen, I looked at your change and I think the right fix is to replace `extern __declspec(selectany)` with `inline`.  In this context, `inline` means, essentially, if there would be multiple definitions of a variable or function marked inline, the linker should use just one definition.  This is equivalent to the intended behavior with your change.  I ran a diff between all three options and here are size results:
- Release build, master branch: `4,150,272` bytes
- Release build, extern __declspec(selectany): `4,147,712` bytes
- Release build, inline: `4,147,712` bytes

As you can see, the difference in size reduction is the same (Please verify for yourself).

Given that the size reduction is the same, we should prefer `inline` because it is standard C++ and avoids any issues later, such as using a macro for `__declspec(selectany)`.

Reply:
Thanks for taking a look at the changes @danbelcher-MSFT!

I rebuilt using `inline` instead of `extern __declspec(selectany)` and I get the same results when examining the output from the linker. (Only one copy of the object instead of 11).

I've pushed an additional commit with the change. Though I'm not sure if you'd rather I force push a new commit, or perhaps just squash merge the two commits when merging?

Reply:
Don't worry about it.  We're taking responsibility of updating the commit message when we close the PR out.  Thanks for the consideration though!